tests/unit_tests/libs/controllers/test_predictor.py

```diff
@@ -714,3 +714,18 @@ def test_user_provided_split_indices(self):
         assert set(predictor.transaction.input_data.train_df['col_a'].tolist()) == set(range(30, 60))
         assert set(predictor.transaction.input_data.test_df['col_a'].tolist()) == set(range(60, 100))
         assert set(predictor.transaction.input_data.validation_df['col_a'].tolist()) == set(range(0, 30))
+
+    def test_save_load_model(self):
+        name = 'saveLoadTest'
+        p = Predictor(name=name)
+        p.learn(from_data="https://s3.eu-west-2.amazonaws.com/mindsdb-example-data/home_rentals.csv",
+                to_predict='rental_price',
+                stop_training_in_x_seconds=3,
+                advanced_args={'force_predict': True})
+
+        p = None
+        F.export_predictor(name)
+        F.import_model(f"{name}.zip")
+
+        p = Predictor(name=name)
+        predictions = p.predict(when_data={'sqft': 1000}, run_confidence_variation_analysis=True)
```

Comment:
Training a model here wastes precious time, please combine it with one of the other tests that trains a model and make a comment as to what it does.